### DISCUSSION TITLE
Creating executable binaries for linux and Mac.

### DISCUSSION THREAD

**Discussion by amalthomas-exe** [ThioJoe/YT-Spammer-Purge#41] [Ideas]
Was wondering if you were planning to to create executable binaries for linux and Mac.

Ant btw, how have you converted the .py to an exe file ?

**Comment by Wind-Explorer** [comment]
I assume you can compile the script by yourself with 'python -m pycompile path/to/script.py' then you can get a compiled python script with .pyc extension which can be executed on any Mac or Linux machine with python installed

**Comment by ageekhere** [comment]
@amalthomas-exe I use https://pypi.org/project/auto-py-to-exe/

**Comment by Wind-Explorer** [comment]
> @amalthomas-exe I use https://pypi.org/project/auto-py-to-exe/

but that creates a .exe file that can only be executed on Microsoft Windows which defeats the purpose of this issue

**Comment by amalthomas-exe** [comment]
> > @amalthomas-exe I use https://pypi.org/project/auto-py-to-exe/
> 
> but that creates a .exe file that can only be executed on Microsoft Windows which defeats the purpose of this issue

Well we can take a step backward and use pyinstaller, on which auto-py-to-exe is based. But the problem is the size of the resulting executable will be large.

**Comment by ageekhere** [comment]
@Wind-Explorer I guess you could use
Portable PyPy for linux
py2app for mac
